client/transactions/index.tsx

```diff
@@ -32,6 +35,7 @@ export const TransactionsPage = () => {
 						<TransactionsList />
 					</>
 				}
+				loadingExperience={ null }
```

Comment:
I'm also forced to specify this, why ?!

Reply:
Just needed to fix the declaration a bit :)